### PR TITLE
LTI Certification 1.1 - Tests 1.5, 1.6 & 1.7

### PR DESCRIPTION
**Pull request by jon-betts**
Fixing a number of issues around the `lti_version` which can only be one value for LTI 1.1.

Addresses: 
* https://github.com/hypothesis/lms/issues/1113 - LTI version invalid
* https://github.com/hypothesis/lms/issues/1114 - LTI version wrong
* https://github.com/hypothesis/lms/issues/1117 - LTI version missing

### CODE REVIEW

File: lms/validation/_lti_launch_params.py [hunk 1]
```diff
@@ -17,14 +18,20 @@ class LaunchParamsSchema(PyramidRequestSchema):
...
+        class Meta:
+            """Allow other values, as we are only here to validate the URL."""
+
+            unknown = EXCLUDE
+
         launch_presentation_return_url = fields.URL()
 
     resource_link_id = fields.Str(required=True)
     launch_presentation_return_url = fields.Str()
+    lti_version = fields.Str(validate=OneOf(["LTI-1p0"]), required=True)
```
Comment (jon-betts) [inline lms/validation/_lti_launch_params.py]: One of... this exact value

File: tests/functional/lti_certification/test_lti_certification_1_1.py [hunk 2]
```diff
@@ -54,9 +80,12 @@ def assert_response_is_html(cls, response):
         assert response.headers["Content-Type"] == Any.string.matching("^text/html")
         assert response.html
 
+    @classmethod
     def assert_redirected_to_tool_with_message(
-        self, response, lti_params, message=Any.string()
+        cls, app, lti_params, message=Any.string()
     ):
+        response = cls.lti_launch(app, lti_params, status=302)
```
Comment (jon-betts) [inline tests/functional/lti_certification/test_lti_certification_1_1.py]: Cut down some repetition. All the tests do this

File: lms/validation/_lti_launch_params.py (hunk 1 above)
Comment (seanh) [reply lms/validation/_lti_launch_params.py]: Yeah it's amazing. And we're required to fail if it's not there

**Comment by jon-betts** [comment]
I've just run through this in the tool. Looks good.

File: tests/unit/lms/validation/_lti_launch_params_test.py [hunk 3]
```diff
@@ -34,7 +34,18 @@ def test_ValidationError_raised_when_res_link_missing(self, pyramid_request):
...
+    def test_ValidationError_raised_when_res_link_missing_with_bad_return_url(
+        self, pyramid_request
+    ):
+        pyramid_request.params.pop("resource_link_id")
+        pyramid_request.params["launch_presentation_return_url"] = "broken"
+
+        schema = LaunchParamsSchema(pyramid_request)
+
+        with pytest.raises(ValidationError):
+            schema.parse()
```
Comment (seanh) [inline tests/unit/lms/validation/_lti_launch_params_test.py]: You _could_ check that the validation error is as expected here.

```python
with pytest.raises(ValidationError) as exc_info:
    schema.parse()

exc = exc_info.value
# Do assertions about exc.messages dict
```

**Pull request merged**